src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -181,6 +183,7 @@ public static void main(final String[] args) {
     showMainFrame();
     new Thread(() -> setupLogging(GameMode.SWING_CLIENT)).start();
     HttpProxy.setupProxies();
+    checkLocalSystem();
```

Comment:
It probably makes sense to do it in the background. Verifying the network timeout is reasonable too, in case we ever do have to wait for it though, seems like it could be very noticeable.